### DISCUSSION TITLE
problem whith make archive for testflight upload

### DISCUSSION THREAD

**Discussion by boshetyn** [Artificial-Pancreas/iAPS#298] [Q&A]
when try make archive i have error in the end of process

![error](https://github.com/Artificial-Pancreas/iAPS/assets/74248125/f8304239-c2aa-42e5-85e4-bcff58e5b82c)

**Comment by boshetyn** [comment]
UPD

its problem in xcode 15

xcode 14 worked

**Reply by Jon-b-m** [reply]
Fixed in dev branch

**Reply by boshetyn** [reply]
waiting master release :)
